ideascube/tests/test_settings.py

```diff
@@ -8,7 +8,7 @@ def test_all_settings(request):
     assert confdir.check(dir=True)
 
     for path in confdir.listdir():
-        if path.ext != 'py':
+        if path.ext != '.py':
```

Comment:
Hu, does it mean that no setting files were test before ?!
@bochecha did well to ask for fixing tests.